project/Build.scala

```diff
@@ -172,7 +188,7 @@ object Util extends Build {
     name := "util-collection",
     libraryDependencies ++= Seq(
       guavaLib,
-      "org.scalacheck"          %% "scalacheck"          % "1.12.2" % "test"
+      "org.scalacheck" %% "scalacheck" % "1.13.1" % "test"
```

Comment:
instead of upgrading, can we use a different version between 2.11 and 2.12?  We have better tooling internally to make sure things don't break, but we're still on 1.12 internally.  It'll be easier to make sure things still work for 2.11 (which, frankly, we care a lot more about than 2.12 until it's stable) if we keep the versions aligned with what we use inside of twitter.